pyproject.toml

```
@@ -7,7 +7,7 @@ authors = [
 ]
 description = "Python toolkit for analysis, visualization, and comparison of spike sorting output"
 readme = "README.md"
-requires-python = ">=3.9,<4.0"
+requires-python = ">=3.9,<3.14"  # Only numpy 2.0 supported on python 3.12 for windows. We need to wait for fix on neo
```

```suggestion
requires-python = ">=3.9,<3.13"  # Only numpy 2.0 supported on python 3.12 for windows. We need to wait for fix on neo
```

This still allows 3.13 :)

---

Thanks, silly mistake of mine!